src/data/mappings/NewsStoryPageMapping.ts

```diff
@@ -0,0 +1,26 @@
+import { NewsStoryPageProps } from "@/types/index"
```

Comment:
This file is responsible for doing mappings from our data source to our 'global' output structure. Note that the mapping function explicitly expects a Drupal data structure as input, and it explicitly outputs the structure `NewsStoryPageProps`, which is also what the component is required to implement.

Reply:
that makes sense to me, especially w/r/t paragraphs